sdk/apimanagement/arm-apimanagement/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/arm-apimanagement",
   "author": "Microsoft Corporation",
   "description": "ApiManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "5.1.0",
+  "version": "5.2.0",
```

Comment:
is there breaking change requiring major version bump?

Reply:
This actually looks like a patch version bump to me.  There are no additive changes to the API surface, only documentation and parameter validation removals.  I would set the version to `5.1.1` before merging.